[DOCU-857] Updated agile changes  https://community.openproject.org/w… (#24470)
opf/openproject: docs/user-guide/backlogs-scrum/README.md

Depending on the sprint status, either a **Start sprint** (for sprints in planning) or **Complete sprint** (for active sprints) button will be displayed.

> [!NOTE]

> > The **Start sprint** button is disabled by default if another sprint is already active or if no sprint dates have been defined. The button might also not exist if you lack the permission to "Start/complete sprints". This can be changed in [project backlog settings ](../projects/project-settings/backlogs-settings)to allow multiple active sprints.

Reviewer: ```suggestion
> The **Start sprint** button is disabled by default if another sprint is already active or if no sprint dates have been defined. It might also not be available if you do not have the **Start/complete sprints** permission.
> By default, only one sprint can be active at a time. If your project is configured to allow multiple active sprints in the [project backlog settings](../projects/project-settings/backlogs-settings)](../projects/project-settings/backlogs-settings), the **Start sprint** button remains available for additional sprints.
```

Your sprint is set in motion by clicking the **Start sprint** button in the sprint header. Clicking it will open the sprint board. 

> [!NOTE]

> > If multiple active sprints are not activated in your project, you cannot start a new sprint while another sprint is already in progress. In this case, the **Start sprint** button will be disabled. See [project backlog settings](../projects/project-settings/backlogs-settings).

Reviewer: ```suggestion
> If your project is not configured to allow multiple active sprints, you cannot start a new sprint while another sprint is already in progress. In this case, the **Start sprint** button is disabled. To allow multiple active sprints, see [project backlog settings](../projects/project-settings/backlogs-settings)](../projects/project-settings/backlogs-settings).
```
